master/buildbot/worker/docker.py

```diff
@@ -156,12 +156,15 @@ def _thd_start_instance(self, image):
                 'Image "%s" not found on docker host.' % image
             )
 
+        host_conf = docker_client.create_host_config(binds=self.binds)
```

Comment:
I think you intended to include only single commit with `appveyor.yml` changes in this PR?